### PR TITLE
Add Auto Scaling section.

### LINKED ISSUE #18: Cover autoscaling groups

### PR DESCRIPTION
**Pull request by ThanosBaskous**
Resolves #18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -617,7 +619,20 @@ We cover security basics first, since configuring user accounts is something you
...
+## Auto Scaling
 
+### Basics
+
+* Auto Scaling Groups are used to control the number of instances in a service.
+* They can be configured, through “[Scaling Policies](http://docs.aws.amazon.com/autoscaling/latest/userguide/policy_creating.html),” to automatically increase or decrease instance counts based on metrics like CPU utilization, or based on a schedule.
+
+### Tips
+
+* Better matching your cluster size to your current resource requirements through use of Auto Scaling Groups can result in significant cost savings for many types of workloads.
```
Comment (jlevy) [inline README.md]: Costs emoji. 💸

File: README.md [hunk 2]
```diff
@@ -617,7 +619,20 @@ We cover security basics first, since configuring user accounts is something you
...
 
+## Auto Scaling
 
+### Basics
+
+* Auto Scaling Groups are used to control the number of instances in a service.
```
Comment (jlevy) [inline README.md]: For consistency, link/boldface/abbreviation:

[**Auto Scaling Groups**](https://aws.amazon.com/autoscaling/) (ASGs) are...

Comment (jlevy) [inline README.md]: Maybe add something like: "... reducing manual effort to provision or deprovision EC2 instances."

**Comment by jlevy** [comment]
👍 Awesome. Couple notes, otherwise LGTM.

For this PR or future: In the basics section, it could be good to mention the types of auto scaling, ie manual vs scheduled vs dynamic (and ideally which to prefer when). Also, are there any "must read" blog posts or resources around ASGs we should work in?

**Comment by ThanosBaskous** [comment]
Thanks @jlevy, I agree with your feedback and have incorporated it.

**Pull request merged**